.circleci/config.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:8.11.3
+    steps:
+      - checkout 
+      - run:
+          name: Update npm
+          command: 'sudo npm install -g npm@latest'
+      - run:
+          name: Install Node-RED
+          command: sudo npm install -g --unsafe-perm node-red
```

Comment:
You don't actually need to install node red. Its just the platform and isn't used for the actual build.